src/Adapter/OrderState/CommandHandler/AddOrderStateHandler.php

```diff
@@ -72,6 +72,8 @@ private function fillOrderStateWithCommandData(OrderState $orderState, AddOrderS
         $orderState->shipped = $command->isShipped();
         $orderState->paid = $command->isPaid();
         $orderState->delivery = $command->isDelivery();
-        $orderState->template = $command->getLocalizedTemplates();
+        if ($command->isSendEmailEnabled()) {
+            $orderState->template = $command->getLocalizedTemplates();
+        }
```

Comment:
Should the value be forced to empty array in this case?
```suggestion
        $orderState->template = $command->isSendEmailEnabled() ? $command->getLocalizedTemplates() : [];
```